mpc/mpc-aio/src/protocol/garble/exec/dual.rs

```diff
@@ -209,6 +209,56 @@ where
 
         Ok(gc_evaluated.into_summary())
     }
+
+    /// Execute dual execution protocol without the equality check
+    ///
+    /// This can be used when chaining multiple circuits together. Neither party
+    /// reveals the output label decoding.
```

Comment:
I think we should only use `decoding` in names but when explaining in comments we should always say "decoding information".  We use the `decoding` shorthand internally but outside of our circle this will not be understood.